history.rst

```diff
@@ -8,9 +8,11 @@ OMERO version history
 5.4.10 (November 2018)
 ----------------------
 
-This is a bug-fix release.
+This release focuses on improving the Desktop client to ease integration with 3rd party applications e.g. ImageJ/Fiji. The Desktop client is now compatible for offline import using `ome-smuggler <https://github.com/openmicroscopy/omero-ms-queue/>`_.
```

Comment:
The "The Desktop client is now compatible for offline import using ome-smuggler." is a bit awkward. I'm playing with ideas like,

> The Desktop client can now do offline imports using ome-smuggler.

> The Desktop client now supports ome-smuggler for offline imports.

... or somesuch, not too sure really.